src/azure-cli/service_name.json

```diff
@@ -441,48 +446,48 @@
   },
   {
     "Command": "az servicebus",
-    "AzureServiceName": "Azure Service Bus Messaging",
-    "URL": "https://docs.microsoft.com/azure/service-bus-messaging/"
+    "AzureServiceName": "Service Bus Messaging",
+    "URL": "https://learn.microsoft.com/azure/service-bus-messaging"
   },
   {
     "Command": "az sf",
-    "AzureServiceName": "Azure Service Fabric",
-    "URL": "https://docs.microsoft.com/azure/service-fabric/"
+    "AzureServiceName": "Service Fabric",
+    "URL": "https://learn.microsoft.com/azure/service-fabric"
   },
   {
     "Command": "az sig",
-    "AzureServiceName": "Azure Virtual Machines",
-    "URL": "https://docs.microsoft.com/azure/virtual-machines/windows/shared-image-galleries"
+    "AzureServiceName": "Virtual Machines",
+    "URL": "https://learn.microsoft.com/azure/virtual-machines/windows/shared-image-galleries"
   },
   {
     "Command": "az signalr",
-    "AzureServiceName": "Azure SignalR",
-    "URL": "https://docs.microsoft.com/azure/azure-signalr/"
+    "AzureServiceName": "SignalR",
+    "URL": "https://learn.microsoft.com/azure/azure-signalr"
   },
   {
     "Command": "az snapshot",
-    "AzureServiceName": "Azure Storage",
-    "URL": "https://docs.microsoft.com/azure/virtual-machines/windows/managed-disks-overview"
+    "AzureServiceName": "Snapshots",
```

Comment:
Should it belong to "Virtual Machine"?

Reply:
@jsntcy, I took it out of `Storage` because the short-summary reads _Manage point-in-time copies of managed disks, native blobs, or other snapshots._ implying that snapshots can be created for more than VMs.